src/main/java/mops/domain/models/datepoll/DatePollOption.java

```diff
@@ -10,18 +10,15 @@
 class DatePollOption implements ValidateAble {
 
     private final Timespan lifeCycle;
```

Comment:
Wieso gehört zur DatePollOption nochmal eine Timespan? Bzw. wieso ist die Abstimmungszeitspanne nicht in den Metainformationen o.Ä.?

Reply:
die Timespan ist hier der von den Nutzern Vorgeschlagene Zeitraum des Treffens. Ich gebe dir aber recht, ein sinnvollerer Name würde aber Sinn ergeben.